src/psyclone/psyir/tools/dependency_tools.py

```diff
@@ -54,19 +54,28 @@ class DependencyTools(object):
     useful for the user to see.
 
     :param loop_types_to_parallelise: A list of loop types that will be \
-                considered for parallelisation. An example loop type might be\
-                'lat', indicating that only loops over latitudes should be\
-                parallelised. The actually supported list of loop types is\
-                specified in the PSyclone config file. This can be used to\
-                exclude for example 1-dimensional loops.
+        considered for parallelisation. An example loop type might be\
+        'lat', indicating that only loops over latitudes should be\
+        parallelised. The actually supported list of loop types is\
+        specified in the PSyclone config file. This can be used to\
+        exclude for example 1-dimensional loops.
     :type loop_types_to_parallelise: list of str
+    :param language_writer: a backend visitor to convert PSyIR expressions \
+        to a representation in the selected language. This is used for \
+        creating error and warning messages.
+    :type language_writer: None (default is Fortran), or an instance \
+        of :py:class:`psyclone.psyir.backend.visitor.PSyIRVisitor`
     '''
-
-    def __init__(self, loop_types_to_parallelise=None):
+    def __init__(self, loop_types_to_parallelise=None,
+                 language_writer=None):
```

Comment:
Again, I think `language_writer=FortranWriter()` here is fine.

Reply:
It worked here  (or the other way round) - it worked in one place and I've replaced it.